server/src/main/java/io/crate/expression/reference/doc/lucene/LuceneReferenceResolver.java

```diff
@@ -87,6 +87,12 @@ public LuceneCollectorExpression<?> getImplementation(final Reference ref) {
                 }
                 throw new UnsupportedFeatureException("_raw expression does not support subscripts: " + column);
 
+            case DocSysColumns.Names.RAW_CONVERTED:
```

Comment:
Given the `_doc::string` change, this is now no longer required, right?

Reply:
Yes I will completely drop the commit introducing this system column.